cfpb/consumerfinance.gov
# Paying for College - Financial Path to Graduation bug fixes

This PR fixes two bugs with the Paying for College tool:

## Changes
- Fix bug which caused monthly loan debt payment to not be included in total expenses
- Fix Plus loan fields so that rates and fees are static, not editable

## How to test this PR
1. Pull down this branch and get it running!
2. Browse to http://0.0.0.0:8000/paying-for-college/your-financial-path-to-graduation/
3. Check:
    - the monthly expenses totaling in the "Can I Afford My Payment?" section
    - whether you can edit the Plus loan rate and fee fields

## Checklist
